package.json

```
@@ -4,9 +4,12 @@
   "description": "Типотека",
   "main": "./src/service/service.js",
   "scripts": {
-    "test": "npm run eslint",
-    "eslint": "eslint --ext .js src/",
-    "start": "node ./src/service/service.js",
+    "test": "npm run test::rest",
+    "eslint": "npm run eslint && npm run jest",
+    "jest": "jest",
+    "test::rest": "jest .e2e.test.js",
+    "start": "cross-env NODE_ENV=production node ./src/service/service.js",
+    "start::debug": "cross-env NODE_ENV=development node ./src/service/service.js",
     "server": "nodemon ./src/service/service.js --server",
```

это наверное мое упущение в прошлый раз, но свои параметры в npm передаются после `--`
```suggestion
    "server": "nodemon ./src/service/service.js -- --server",
```